lib/sorbet-rails/gem_plugins/flag_shih_tzu_plugin.rb

```diff
@@ -0,0 +1,183 @@
+                                    ],
+                                    returns: 'T::Hash[String, T::Hash[Symbol, T.untyped]]')
+
+    if @model_class.flag_mapping.present?
+      # https://github.com/pboling/flag_shih_tzu#support-for-manually-building-conditions
+      @model_class
+        .flag_mapping
+        .flat_map { |_, flags| flags.keys }
+        .each do |flag_key|
+          next unless really_has_the_flag?(flag_key)
+
+          custom_module_rbi.create_method(
+            "#{flag_key}_condition",
+            parameters: [
+              ::Parlour::RbiGenerator::Parameter.new(
+                'options',
+                type: 'T.nilable(T::Hash[Symbol, T.untyped])'
+              )
+            ],
+            returns: 'String'
+          )
+
+          custom_module_rbi.create_method(
+            "not_#{flag_key}_condition",
+            parameters: [
+              ::Parlour::RbiGenerator::Parameter.new(
+                'options',
+                type: 'T.nilable(T::Hash[Symbol, T.untyped])'
+              )
+            ],
+            returns: 'String'
+          )
+        end
+    end
+  end
+
+  # https://github.com/pboling/flag_shih_tzu#generated-boolean-patterned-instance-methods
+  sig { params(column: String, custom_module_rbi: Parlour::RbiGenerator::ModuleNamespace).void }
+  def add_methods_for_column(column, custom_module_rbi)
+    custom_module_rbi.create_method("all_#{column}",
+                                    returns: 'T::Array[Symbol]')
+
+    custom_module_rbi.create_method("selected_#{column}",
+                                    returns: 'T::Array[Symbol]')
+
+    custom_module_rbi.create_method("select_all_#{column}",
+                                    returns: 'T::Array[Symbol]')
+
+    custom_module_rbi.create_method("unselect_all_#{column}",
+                                    returns: 'T::Array[Symbol]')
+
+    custom_module_rbi.create_method("selected_#{column}=",
+                                    parameters: [
+                                      ::Parlour::RbiGenerator::Parameter.new(
+                                        'chosen_flags',
+                                        type: 'T.nilable(T::Array[Symbol])'
+                                      )
+                                    ],
+                                    returns: 'T::Array[Symbol]')
+
+  end
+
+  # https://github.com/pboling/flag_shih_tzu#generated-boolean-patterned-instance-methods
+  sig { params(column: String, flag_key: Symbol, custom_module_rbi: Parlour::RbiGenerator::ModuleNamespace).void }
+  def add_methods_for_flag(column, flag_key, custom_module_rbi)
+    custom_module_rbi.create_method(flag_key.to_s, returns: 'T::Boolean')
+    custom_module_rbi.create_method("#{flag_key}?", returns: 'T::Boolean')
+    custom_module_rbi.create_method(
+      "#{flag_key}=",
+      parameters: [
+        ::Parlour::RbiGenerator::Parameter.new(
+          'value',
+          type: 'T::Boolean'
```

Comment:
Another comment on style is using comma consistently.

Reply:
Sorry, I don't get this, can you elaborate?